client/modules/core/__tests__/main.js

```diff
@@ -0,0 +1,90 @@
+import { expect } from "meteor/practicalmeteor:chai";
```

Comment:
I am unable to even run this test at the moment. `npm test` is not playing nicely with babel, or something. I have spent many hours trying to get these tests to run with no luck. I would appreciate all suggestions.
to repro, just run `npm test`